README.md

```diff
@@ -1,8 +1,13 @@
 # recOrder
+[![Python package index](https://img.shields.io/pypi/v/waveorder.svg)](https://pypi.org/project/recOrder-napari)
+[![PyPI monthly downloads](https://img.shields.io/pypi/dm/waveorder.svg)](https://pypistats.org/packages/recOrder-napari)
```

Comment:
These are waveorder URLs.

Reply:
Pardon me...sloppy. Thanks for checking @ziw-liu.